### PR TITLE
[CFE-524] Fix - asset selection bug

### PR DESCRIPTION
**Pull request by evilpeach**
- Can't select attach funds
- When adding more assets, a selected asset has gone.

### CODE REVIEW

**Review by Poafs1 — APPROVED** [review]
lgtm, can wait for @songwongtp to approve as well.

File: src/lib/components/fund/selectFund.tsx
Comment (songwongtp) [inline src/lib/components/fund/selectFund.tsx:1]: capitalize filename

File: src/lib/components/fund/selectFund.tsx [hunk 1]
```diff
@@ -62,7 +62,6 @@ export const SelectFund = ({
             setValue(`${ASSETS_SELECT}.${idx}.denom`, newVal)
           }
           assetOptions={assetOptions}
-          initialSelected={field.denom}
```
Comment (songwongtp) [inline src/lib/components/fund/selectFund.tsx:65]: not working

**Review by songwongtp — CHANGES_REQUESTED** [review]

**Review by Poafs1 — APPROVED** [review]
lgtm

File: src/lib/components/forms/SelectInput.tsx [hunk 2]
```diff
@@ -81,11 +81,13 @@ export const SelectInput = <T extends string>({
...
+    if (!initialSelected) return;
+
     setSelected(
-      options.find((item) => !item.disabled && item.value === initialSelected)
-        ?.label ?? ""
+      options.find((item) => item.value === initialSelected)?.label ?? ""
     );
-  }, [initialSelected, options]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [initialSelected]);
```
Comment (evilpeach) [inline src/lib/components/forms/SelectInput.tsx:90]: Revert this. It's no need to remove options

File: src/lib/components/fund/SelectFund.tsx [hunk 3]
```diff
@@ -75,6 +74,7 @@ export const SelectFund = ({
               labelBgColor={labelBgColor}
             />
           }
+          initialSelected={field.denom}
```
Comment (evilpeach) [inline src/lib/components/fund/SelectFund.tsx:77]: revert this.

**Review by songwongtp — APPROVED** [review]

**Pull request merged**